packages/jss/src/index.js

```diff
@@ -7,8 +7,27 @@
  * @license MIT
  */
 import Jss from './Jss'
+import StyleSheet from './StyleSheet'
 import type {JssOptions} from './types'
 
+/**
+ * Export types for better typing inside plugins and integrations.
+ */
+export type {
+  StyleSheetOptions,
+  JssValue,
+  JssOptions,
+  JssStyle,
+  Plugin,
+  generateClassName,
+  RuleListOptions,
```

Comment:
Is this one needed somewhere?

Reply:
Maybe, I don't know. It might be needed at some time especially because we export RuleList.
Also, I would like to export those now and not have to add exports every time later on.